### DISCUSSION TITLE
Custom shortcut

### DISCUSSION THREAD

**Discussion by Java-soft** [dbeaver/dbeaver#11319] [Q&A] [answered]
When I execute SQL, I hope it is Command + enter, but now it is control + enter, so it is very uncomfortable to use

**Comment by uslss** [comment, marked as answer]
Shortcuts already could be customized in Preferences
![image](https://user-images.githubusercontent.com/31996417/96422447-b28fca80-1200-11eb-9959-a9e41c6f9eb5.png)
